### DISCUSSION TITLE
Substitution plastique

### DISCUSSION THREAD

**Discussion by valeriemerle** [betagouv/ma-cantine#1127] [Autres sujets]
Réemploi – 6 grandes cuisines centrales créent le groupement de commandes «Tremplin » : le Siresco (93), le Sivu Bordeaux – Mérignac (33), le Syrec (92), les villes de Nantes (44), Rennes (35) et Toulouse (31) annoncent la création  du groupement de commandes « Tremplin ». Forts de 150 000 repas servis par jour, ses 6 membres plaident pour une collaboration inédite avec  industriels et fournisseurs de solutions alternatives auxquels le groupement peut permettre d’optimiser leurs offres aux collectivités en fonction du volume de commande prévisionnel et de mieux amortir leurs propres investissements en R&D.
Voir l'article : https://www.restauration21.fr/restauration21/2022/02/r%c3%a9emploi-6-grandes-cuisines-centrales-cr%c3%a9ent-le-groupement-de-commandes-tremplin.html

Portage à domicile:
Les services de restauration à domicile qui proposent un abonnement à des prestations de repas préparés et livrés au moins 4 fois par semaine utilisent de la vaisselle, des couverts et des récipients de transport des aliments et des boissons réemployables. Ils doivent également procéder à leur collecte en vue de leur réemploi.

**Comment by hfroot** [comment]
@valeriemerle est-ce qu'il y a qqch à faire sur le site avec ça ?
